src/mocks/handlers.ts

```diff
@@ -21,6 +22,46 @@ export const handlers = [
     );
   }),
 
+  /**
+   * Build a fake ensure user function
+   * Currently will ALWYAS return Brenda Wedding
+   * To update this, we'll need to track users and user ID's
+   */
+  rest.post("/_api/web/ensureuser", (req, res, ctx) => {
+    return res(
+      ctx.status(200),
+      ctx.delay(responsedelay),
+      ctx.json({
+        Id: 14,
+        IsHiddenInUI: false,
+        LoginName: "i:0#.f|membership|brenda.wedding@us.af.mil",
+        Title: "WEDDING, BRENDA K NH-04 USAF AFMC AFLCMC/OZIC",
+        PrincipalType: 1,
+        Email: "brenda.wedding@us.af.mil",
+        Expiration: "",
+        IsEmailAuthenticationGuestUser: false,
+        IsShareByEmailGuestUser: false,
+        IsSiteAdmin: false,
+        UserId: {
+          NameId: "1003000099a500ad",
+          NameIdIssuer: "urn:federation:microsoftonline",
+        },
+        UserPrincipalName: "brenda.wedding@us.af.mil",
+      })
+    );
+  }),
+
+  /**
+   * Build emails API
+   */
+  rest.post("/_api/web/lists/getByTitle\\('Emails')/items", (req, res, ctx) => {
```

Comment:
Should we post something to the console, with our Email message -- so we know it was generated?

Reply:
Ooops, don't know why I was thinking this -- the service worker posts it for us.  Maybe you can show me if there is a trick to actually viewing the data the MSW outputs.